src/applications/disability-benefits/2346/containers/OrderHistory.jsx

```diff
@@ -0,0 +1,228 @@
+                    <tr key={dlcUIObj.id}>
+                      <td>{dlcUIObj.dateProcessed}</td>
+                      <td>{dlcUIObj.productGroup}</td>
+                      <td>{dlcUIObj.product}</td>
+                      <td>{dlcUIObj.id}</td>
+                      <td>{dlcUIObj.orderIntakeMethod}</td>
+                      <td>{dlcUIObj.checkbox} </td>
+                    </tr>
+                  ))}
+                </tbody>
+              </table> */
+              }
+              <table id="accessoriesTable" className="form2346 vads-c-table">
+                <tbody>
+                  <tr>
+                    <th className="form2346 vads-c-table__th">
+                      Date Processed
+                    </th>
+                    <th className="form2346 vads-c-table__th">Product Type</th>
+                    <th className="form2346 vads-c-table__th">Model</th>
+                    <th className="form2346 vads-c-table__th">Serial Number</th>
+                    <th className="form2346 vads-c-table__th">Order Method</th>
+                    <th className="form2346 vads-c-table__th--narrow vads-u-text-align--center">
+                      Check to Order Accessories
+                    </th>
+                  </tr>
+                  {dlcUIAccessoriesData.map(dlcUIObj => (
+                    <tr key={dlcUIObj.id}>
+                      <td>{dlcUIObj.dateProcessed}</td>
+                      <td>{dlcUIObj.productGroup}</td>
+                      <td>{dlcUIObj.product}</td>
+                      <td>{dlcUIObj.id}</td>
+                      <td>{dlcUIObj.orderIntakeMethod}</td>
+                      <td>{dlcUIObj.checkbox} </td>
+                    </tr>
+                  ))}
+                </tbody>
+              </table>
+              ={' '}
+              {/* <button
+                type="button"
+                className="usa-button-primary va-button-primary"
+              >
+                Back
+              </button>
+              <button
+                type="button"
+                className="usa-button-primary va-button-primary"
+              >
+                Submit
+              </button> */}
+            </>
+          )}
+      </div>
+    );
+  }
+}
+
+const mapStateToProps = state => ({
+  dlcBatteryData: state?.form2346Reducer?.dlcBatteryData || '',
+  // NOTE: Decision was to take socks out of MVP -@maharielrosario at 1/28/2020, 9:49:20 AM
+  // dlcSocksData: state?.form2346Reducer?.dlcSocksData || '',
+  dlcAccessoriesData: state?.form2346Reducer?.dlcAccessoriesData || '',
+  firstName: state?.user?.profile?.userFullName?.first || '',
+  lastName: state?.user?.profile?.userFullName?.last || '',
+  address: state?.user?.profile?.vet360?.mailingAddress?.addressLine1 || '',
+  city: state?.user?.profile?.vet360?.mailingAddress?.city || '',
+  state: state?.user?.profile?.vet360?.mailingAddress?.stateCode || '',
+  zip: state?.user?.profile?.vet360?.mailingAddress?.zipCode || '',
```

Comment:
^^ https://github.com/department-of-veterans-affairs/vets-website/pull/11679/files#r377901915

Reply:
This has been corrected.